modules/customize-certificates-rotate-service-serving.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/service-serving-certificate.adoc
+
+[id="rotate-service-serving-{context}"]
+= Manually rotate the generated service certificate
+
```

Comment:
You need an intro, maybe, "If you /<reason why you'd do this\>, you must manually rotate the generated service certificate before it expires. \<dire warning about what happens if you don't.\>"

Reply:
I've included a bit more information about what happens when it occurs. I don't have a strong "why" as of now, but I can include this in a separate PR once it's available.